habitat_code/example-big-sur.py

```diff
@@ -0,0 +1,117 @@
+# %% [markdown]
+# # Changes in vegetation caused by the Mud Creek Landslide
+# 
+# ![Mud Creek Landslide](https://d9-wret.s3.us-west-2.amazonaws.com/assets/palladium/production/s3fs-public/styles/list_item/public/thumbnails/image/BigSur1_May20SlideDES_0.jpg)
+# 
+# > Image source: [USGS](https://www.usgs.gov/special-topics/big-sur-landslides/science/mud-creek-landslide-may-20-2017)
+
+# %%
+import re
+
+import earthaccess
+import pandas as pd
+import rioxarray as rxr
+
+# %%
+# Location and time information for the Mud Creek Landslide
+lat = 35.76
+lon = -121.30
+min_lat = lat - .5
+max_lat = lat + .5
+min_lon = lon - .5
+max_lon = lon + .5
+bbox = (min_lon, min_lat, max_lon, max_lat)
+
+date = '2017-05-20'
+min_date = '2017-03'
+max_date = '2017-07'
+date_range = (min_date, max_date)
+
+# %%
+earthaccess.login()
+
+# %%
+results = earthaccess.search_data(
```

Comment:
Consider adding some description here so we know why these parameters were chosen.